content/docs/details/custom-api-docs-renderers/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Use custom renderer with your API entities
+publishedDate: '2022-10-28T13:00:00.0Z'
+description: How to use custom plugins to render APIs
+---
+
+## Prerequisites
+
+* You must have custom plugins enabled for your tenant. Contact Roadie to enquire about this.
+
+## Introduction
+
+The API docs plugins supports user supplied renderers for API definitions. You might want to use this feature to render
+a custom format or to make modifications to an existing format. 
```

Comment:
Can you add a sentence a bit like this: "This page explains how you can configure such a user supplied renderer in roadie."